templates/page.tpl.php

```diff
@@ -89,16 +89,22 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
     <div class="wrapper">
       <section class="footer-top">
         <nav>
-          <?php print render($page['footer']); ?>
+          <?php print render($page['footer_top']); ?>
         </nav>
       </section>
+      <section>
+        <div class="footer-logo"></div>
+        <div class="footer-links">
+          <nav>
+            <?php print render($page['footer']); ?>
+          </nav>
```

Comment:
@Feng-Shui how do you envisage the theme settings for vertical / horizontal placement might work in terms of implementation? I'm trying to gauge how it might relate to progressing the card in terms of what the final markup / css would look like to meet our requirements.